sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -1031,16 +1044,21 @@ class RocksDBSuite extends AlsoTestWithChangelogCheckpointingEnabled with Shared
       val fileManager = new RocksDBFileManager(
         dfsRootDir, Utils.createTempDir(), new Configuration)
       val sstDir = s"$dfsRootDir/SSTs"
-      def numRemoteSSTFiles: Int = listFiles(sstDir).length
+      def numRemoteSSTFiles: Int = {
+        val files = listFiles(sstDir)
+        logWarning(s"SST files on DFS: $files")
```

Comment:
nit: probably debug log we could delete as of now? It you find the log to be helpful on debugging, consider using logDebug or logInfo with more verbose log message.

Reply:
If we are going to delete this line, let's just revert to one liner (old one)

Reply:
removed